openlr/openlr_match_quality/assessment_tool/mainwindow.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "openlr/openlr_match_quality/assessment_tool/traffic_mode.hpp"
+
+#include "base/string_utils.hpp"
+
+#include <QMainWindow>
+
+class Framework;
+class TrafficMode;
+
+namespace df
+{
+class DrapeApi;
+}  // namespace df
```

Comment:
Optional: consider getting rid of ```// namespace df```, because usually in headers for forward declarations we omit such comments.